templates/ci/gcc.yml.metadata

```diff
@@ -0,0 +1,3 @@
+{
+  "iconTaskName": ""
```

Comment:
This template is all script steps, and our default `FileCode` icon seemed more appropriate than `CmdLine` to me. We could invest in associating icons directly with templates and have a new endpoint to serve them.